vm/vm.go

```diff
@@ -0,0 +1,121 @@
+package vm
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/kubearmor/kubearmor-client/k8s"
+	pb "github.com/kubearmor/kubearmor-client/protobuf"
+	"github.com/rs/zerolog/log"
+	"google.golang.org/grpc"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type VmOptions struct {
+	IP     string
+	VMName string
+	File   string
+}
+
+var (
+	serviceAccountName = "kvmsoperator"
+	pbClient           pb.HandleCliClient
+)
+
+func initGrpcClient(ip string) error {
+	// Connect to gRPC server
+	grpcClientConn, err := grpc.DialContext(context.Background(), net.JoinHostPort(ip, "32770"), grpc.WithInsecure())
```

Comment:
This PORT should be made configurable using an env var.

Reply:
sure Rahul.

Reply:
Hey @seswarrajan , the comment seems to be marked resolved without the actual fix.